### ISSUE TITLE
The  "Library/share/RDKit" directory is missing in newest conda releases

### ISSUE THREAD

**Issue by kienerj** [conda-forge/rdkit-feedstock#183]
### Solution to issue cannot be found in the documentation.

- [x] I checked the documentation.

### Issue

As the title says. "Library/share/RDKit" directory seems to be missing? This includes Docs, Data and the Contrib folders among others. In environments with 2023.09 or 2024.03 versions, it is still there but in 2024.09 it doesn't exist anymore.

This is on Windows 10.

### Installed packages

# Name                    Version                   Build  Channel
_py-xgboost-mutex         2.0                       cpu_0    conda-forge
asttokens                 2.2.1              pyhd8ed1ab_0    conda-forge
aws-c-auth                0.6.28               habb20d3_5    conda-forge
aws-c-cal                 0.5.27               hc655790_0    conda-forge
aws-c-common              0.8.20               hcfcfb64_0    conda-forge
aws-c-compression         0.2.17               h0a579c1_0    conda-forge
aws-c-event-stream        0.3.0                h92d84d2_6    conda-forge
aws-c-http                0.7.8                haeac8d2_4    conda-forge
aws-c-io                  0.13.26              hf956cbd_0    conda-forge
aws-c-mqtt                0.8.13               h73587ab_2    conda-forge
aws-c-s3                  0.3.4                h26b4012_5    conda-forge
aws-c-sdkutils            0.1.10               h0a579c1_0    conda-forge
aws-checksums             0.1.16               h0a579c1_0    conda-forge
aws-crt-cpp               0.20.2               h0db1998_9    conda-forge
aws-sdk-cpp               1.10.57             ha442f66_14    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                hcfcfb64_8    conda-forge
brotli-bin                1.0.9                hcfcfb64_8    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
c-ares                    1.19.1               hcfcfb64_0    conda-forge
ca-certificates           2024.8.30            h56e8100_0    conda-forge
cairo                     1.18.0               h1fef639_0    conda-forge
certifi                   2024.8.30          pyhd8ed1ab_0    conda-forge
cffi                      1.16.0          py311ha68e1ae_0    conda-forge
charset-normalizer        3.1.0              pyhd8ed1ab_0    conda-forge
clr_loader                0.2.6              pyhd8ed1ab_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
comm                      0.1.3              pyhd8ed1ab_0    conda-forge
contourpy                 1.1.0           py311h005e61a_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
debugpy                   1.6.7           py311h12c1d0e_0    conda-forge
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
docopt                    0.6.2                      py_1    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
expat                     2.5.0                h63175ca_1    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.14.2               hbde0cde_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fonttools                 4.40.0          py311ha68e1ae_0    conda-forge
freetype                  2.12.1               h546665d_1    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
git                       2.41.0               h57928b3_0    conda-forge
glib                      2.78.1               h12be248_0    conda-forge
glib-tools                2.78.1               h12be248_0    conda-forge
greenlet                  2.0.2           py311h12c1d0e_1    conda-forge
gst-plugins-base          1.22.7               h001b923_0    conda-forge
gstreamer                 1.22.7               hb4038d2_0    conda-forge
icu                       73.2                 h63175ca_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.7.0              pyha770c72_0    conda-forge
importlib_metadata        6.7.0                hd8ed1ab_0    conda-forge
intel-openmp              2023.1.0         h57928b3_46319    conda-forge
ipykernel                 6.23.2             pyh6817e22_0    conda-forge
ipython                   8.14.0             pyh08f2357_0    conda-forge
ipywidgets                8.1.0              pyhd8ed1ab_0    conda-forge
jedi                      0.18.2             pyhd8ed1ab_0    conda-forge
joblib                    1.2.0              pyhd8ed1ab_0    conda-forge
jupyter_client            8.2.0              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.1           py311h1ea47a8_0    conda-forge
jupyterlab_widgets        3.0.8              pyhd8ed1ab_0    conda-forge
kiwisolver                1.4.4           py311h005e61a_1    conda-forge
krb5                      1.21.3               hdf4eb48_0    conda-forge
lcms2                     2.15                 h3e3b177_1    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libabseil                 20230125.2      cxx17_h63175ca_2    conda-forge
libarrow                  12.0.0           h5f84808_8_cpu    conda-forge
libblas                   3.9.0              17_win64_mkl    conda-forge
libboost                  1.86.0               h444863b_2    conda-forge
libboost-python           1.86.0          py311h9b10771_2    conda-forge
libbrotlicommon           1.0.9                hcfcfb64_8    conda-forge
libbrotlidec              1.0.9                hcfcfb64_8    conda-forge
libbrotlienc              1.0.9                hcfcfb64_8    conda-forge
libcblas                  3.9.0              17_win64_mkl    conda-forge
libclang                  15.0.7          default_h77d9078_2    conda-forge
libclang13                15.0.7          default_h77d9078_2    conda-forge
libcrc32c                 1.1.2                h0e60522_0    conda-forge
libcurl                   8.10.1               h1ee3ff0_0    conda-forge
libdeflate                1.18                 hcfcfb64_0    conda-forge
libevent                  2.1.12               h3671451_1    conda-forge
libexpat                  2.5.0                h63175ca_1    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libglib                   2.78.1               he8f3873_0    conda-forge
libgoogle-cloud           2.11.0               hec1a1f0_1    conda-forge
libgrpc                   1.55.1               h0c1e3fa_1    conda-forge
libhwloc                  2.9.1           nocuda_h15da153_6    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libjpeg-turbo             2.1.5.1              hcfcfb64_0    conda-forge
liblapack                 3.9.0              17_win64_mkl    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libpng                    1.6.39               h19919ed_0    conda-forge
libpq                     16.4                 hab9416b_2    conda-forge
libprotobuf               4.23.2               h1975477_5    conda-forge
librdkit                  2024.09.1            h66803c0_1    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libsqlite                 3.46.1               h2466b09_0    conda-forge
libssh2                   1.11.0               h7dfc565_0    conda-forge
libthrift                 0.18.1               h06f6336_2    conda-forge
libtiff                   4.5.1                h6c8260b_0    conda-forge
libutf8proc               2.8.0                h82a8f57_0    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp-base              1.3.0                hcfcfb64_0    conda-forge
libxcb                    1.15                 hcd874cb_0    conda-forge
libxgboost                1.7.4            cpu_h20390bd_2    conda-forge
libxml2                   2.11.4               hc3477c8_0    conda-forge
libxslt                   1.1.37               h6070c61_1    conda-forge
libzlib                   1.3.1                h2466b09_1    conda-forge
lxml                      4.9.2           py311h750ae06_1    conda-forge
lz4-c                     1.9.4                hcfcfb64_0    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
matplotlib                3.7.1           py311h1ea47a8_0    conda-forge
matplotlib-base           3.7.1           py311h6e989c2_0    conda-forge
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mkl                       2022.1.0           h6a75c08_874    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
numpy                     1.25.0          py311h0b4df5a_0    conda-forge
openjpeg                  2.5.0                ha2aaf27_2    conda-forge
openssl                   3.3.2                h2466b09_0    conda-forge
orc                       1.8.4                h08d22ad_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.0.2           py311hf63dbb6_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.5.0           py311hde623f7_1    conda-forge
pip                       23.1.2             pyhd8ed1ab_0    conda-forge
pixman                    0.43.4               h63175ca_0    conda-forge
platformdirs              3.6.0              pyhd8ed1ab_0    conda-forge
plotly                    5.15.0             pyhd8ed1ab_0    conda-forge
ply                       3.11                       py_1    conda-forge
pooch                     1.7.0              pyha770c72_3    conda-forge
prompt-toolkit            3.0.38             pyha770c72_0    conda-forge
prompt_toolkit            3.0.38               hd8ed1ab_0    conda-forge
psutil                    5.9.5           py311ha68e1ae_0    conda-forge
psycopg                   3.2.2           py311h9e3867f_0    conda-forge
psycopg-c                 3.2.2           py311h4e9e98d_0    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py-xgboost                1.7.4           cpu_py311h3180b0a_2    conda-forge
py4j                      0.10.9.7           pyhd8ed1ab_0    conda-forge
pyarrow                   12.0.0          py311h6a6099b_8_cpu    conda-forge
pycairo                   1.24.0          py311h99894aa_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pygments                  2.15.1             pyhd8ed1ab_0    conda-forge
pyparsing                 3.1.0              pyhd8ed1ab_0    conda-forge
pyqt                      5.15.7          py311h125bc19_3    conda-forge
pyqt5-sip                 12.11.0         py311h12c1d0e_3    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.11.4          h2628c8c_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.11                    3_cp311    conda-forge
pythonnet                 3.0.3              pyhd8ed1ab_0    conda-forge
pytz                      2023.3             pyhd8ed1ab_0    conda-forge
pywin32                   304             py311h12c1d0e_2    conda-forge
pyyaml                    6.0             py311ha68e1ae_5    conda-forge
pyzmq                     25.1.0          py311h7b3f143_0    conda-forge
qt-main                   5.15.8              he5a7383_16    conda-forge
rdkit                     2024.09.1       py311h9efb452_1    conda-forge
re2                       2023.03.02           hd4eee63_0    conda-forge
reportlab                 3.6.13          py311hae2e3ae_0    conda-forge
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
scikit-learn              1.2.2           py311h142b183_2    conda-forge
scipy                     1.10.1          py311h37ff6ca_3    conda-forge
seaborn                   0.12.2               hd8ed1ab_0    conda-forge
seaborn-base              0.12.2             pyhd8ed1ab_0    conda-forge
setuptools                67.7.2             pyhd8ed1ab_0    conda-forge
sip                       6.7.9           py311h12c1d0e_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
snappy                    1.1.10               hfb803bf_0    conda-forge
sqlalchemy                2.0.16          py311ha68e1ae_0    conda-forge
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
statsmodels               0.14.0          py311h59ca53f_1    conda-forge
tbb                       2021.9.0             h91493d7_0    conda-forge
tenacity                  8.2.2              pyhd8ed1ab_0    conda-forge
threadpoolctl             3.1.0              pyh8a188c0_0    conda-forge
tk                        8.6.12               h8ffe710_0    conda-forge
toml                      0.10.2             pyhd8ed1ab_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
tornado                   6.3.2           py311ha68e1ae_0    conda-forge
tqdm                      4.65.0             pyhd8ed1ab_1    conda-forge
traitlets                 5.9.0              pyhd8ed1ab_0    conda-forge
typing-extensions         4.6.3                hd8ed1ab_0    conda-forge
typing_extensions         4.6.3              pyha770c72_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
urllib3                   2.0.3              pyhd8ed1ab_0    conda-forge
vc                        14.3                hb25d44b_16    conda-forge
vc14_runtime              14.34.31931         h5081d32_16    conda-forge
vs2015_runtime            14.34.31931         hed1258a_16    conda-forge
wcwidth                   0.2.6              pyhd8ed1ab_0    conda-forge
wheel                     0.40.0             pyhd8ed1ab_0    conda-forge
widgetsnbextension        4.0.8              pyhd8ed1ab_0    conda-forge
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
xgboost                   1.7.4           cpu_py311h3180b0a_2    conda-forge
xorg-libxau               1.0.11               hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
yaml                      0.2.5                h8ffe710_2    conda-forge
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.15.0             pyhd8ed1ab_0    conda-forge
zlib                      1.3.1                h2466b09_1    conda-forge
zstd                      1.5.6                h0ea2cb4_0    conda-forge

### Environment info

active environment : knime
    active env location : C:\Users\kienerj\AppData\Local\mambaforge\envs\knime
            shell level : 3
       user config file : C:\Users\kienerj\.condarc
 populated config files : C:\Users\kienerj\AppData\Local\mambaforge\.condarc
          conda version : 23.1.0
    conda-build version : not installed
         python version : 3.10.10.final.0
       virtual packages : __archspec=1=x86_64
                          __win=0=0
       base environment : C:\Users\kienerj\AppData\Local\mambaforge  (writable)
      conda av data dir : C:\Users\kienerj\AppData\Local\mambaforge\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\Users\kienerj\AppData\Local\mambaforge\pkgs
                          C:\Users\kienerj\.conda\pkgs
                          C:\Users\kienerj\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\kienerj\AppData\Local\mambaforge\envs
                          C:\Users\kienerj\.conda\envs
                          C:\Users\kienerj\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.1.0 requests/2.31.0 CPython/3.10.10 Windows/10 Windows/10.0.17763
          administrator : True
             netrc file : None
           offline mode : False

**Comment by skearnes** [comment]
Is there a particular feature you're looking for? We recently updated the conda package to exclude some of the packages you mentioned that weren't deemed critical to the conda package: https://github.com/conda-forge/rdkit-feedstock/blob/main/recipe/install.bat#L6

**Comment by kienerj** [comment]
At least for me the Contrib directory is helpful. Contrib contains contributed code which I from time to time make use of. Not having it by default and downloading it manually is then a chore especially because you want it in-sync with the rdkit release so has to be done for each environment separately. also it's much easier to import if they have a clear relative path from the current environment.

**Comment by skearnes** [comment]
Thanks; I can look into this and see if we can get contrib back into the python package.

**Comment by greglandrum** [comment]
I do think there's some value in having Contrib included with the python packages. It contains useful stuff that some people are definitely using.

**Comment by greglandrum** [comment]
> I do think there's some value in having Contrib included with the python packages. It contains useful stuff that some people are definitely using.

I should have noticed when we took it out and raised an objection, but that set of changes ended up being a bit overwhelming for me.

**Comment by skearnes** [comment]
> > I do think there's some value in having Contrib included with the python packages. It contains useful stuff that some people are definitely using.
> 
> I should have noticed when we took it out and raised an objection, but that set of changes ended up being a bit overwhelming for me.

Looks like this is only a windows issue (the contrib modules are present in the OSX and Linux packages); I'll fix in #184.
